src/System.Console/src/System/Console.cs

```diff
@@ -53,7 +53,7 @@ public static Encoding InputEncoding
 
                     Volatile.Write(ref s_inputEncoding, (Encoding)value.Clone());
 
-                    // We need to reinitialize Console.In in the next call to s_in
+                    // We need to reinitialize Console.In the next call to s_in
```

Comment:
This was not actually a double word.

```suggestion
                    // We need to reinitialize 'Console.In' in the next call to s_in
```